assets/js/translator.js

```diff
@@ -802,6 +925,17 @@ function translationNotAvailable() {
         .addClass('notAvailable text-danger');
 }
 
+function translationNotAvailableWebpage(data) {
+    translationNotAvailable();
+    var div = $('<div id="translatedWebpage" class="translatedWebpage"></div>');
+    div.text(getDynamicLocalization('Not_Available'));
+    div.addClass('notAvailable text-danger');
+    $('#translatedWebpage').replaceWith(div[0]);
+    $('#translatedWebpage').append($('<div></div>').text(' '));
```

Comment:
One `.append` with an array argument with these three should work and be much cleaner? Also can be chained I think.